### PR TITLE
[new release] spawn (0.14.0)

### PR DESCRIPTION
**Pull request by jeremiedimino**
Spawning sub-processes

- Project page: <a href="https://github.com/janestreet/spawn">https://github.com/janestreet/spawn</a>
- Documentation: <a href="https://janestreet.github.io/spawn/">https://janestreet.github.io/spawn/</a>

##### CHANGES:

- Do not use vfork by default on OSX as it seems broken (janestreet/spawn#20,
  @jeremiedimino)

- Fix compilation under msvc (janestreet/spawn#21, @nojb)

### CODE REVIEW

File: packages/spawn/spawn.0.14.0/opam [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+homepage: "https://github.com/janestreet/spawn"
+doc: "https://janestreet.github.io/spawn/"
+bug-reports: "https://github.com/janestreet/spawn/issues"
+depends: [
+  "dune" {>= "2.5"}
+  "ppx_expect" {with-test}
+  "ocaml" {>= "4.02.3"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/spawn/spawn.0.14.0/opam:33]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
Thanks!

**Comment by kit-ty-kate** [comment]
The v prefix was missing from the release :/ I'm not sure what to do it's a bit late for removing the package now but i also don't really want to keep it around. There will be a solution with opam 2.2.0 but in the meantime this will look a bit dirty.

**Comment by mseri** [comment]
Should we re-release it as v0.14.0 and mark this as available:false?

**Comment by kit-ty-kate** [comment]
> Should we re-release it as v0.14.0 and mark this as available:false?

Sadly no, we don't know if anybody use lock files with `0.14.0`. I've opened https://github.com/ocaml/opam-repository/pull/19266 with a fix + comment for when opam 2.2.0 is out

**Pull request merged**